src/Service/Models/RecommendationContext.php

```diff
@@ -5,38 +5,35 @@
 class RecommendationContext
 {
     public function __construct(
```

Comment:
Du kan ikke bare fjerne properties

```suggestion
    public function __construct(
        public ?array $hierarchies = null,
        public ?string $brand = null,
        public ?array $urls = null
```